core/arch/arm/kernel/link.mk

```diff
@@ -117,7 +117,12 @@ endef
 # filter-out to workaround objdump warning
 version-o-cflags = $(filter-out -g3,$(core-platform-cflags) \
 			$(platform-cflags) $(cflagscore))
+# SOURCE_DATE_EPOCH defined for reproducible builds
+ifneq ($(SOURCE_DATE_EPOCH),)
+DATE_STR = `date -u -d @$${SOURCE_DATE_EPOCH}`
```

Comment:
Why isn't `SOURCE_DATE_EPOCH` supplied as `$(SOURCE_DATE_EPOCH)`? I mean what's the purpose of using `@$${SOURCE_DATE_EPOCH}` instead which seems to be at least partially evaluated in the shell.